portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -15,18 +15,41 @@
 package com.google.sps.servlets;
 
 import java.io.IOException;
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Arrays;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet that returns some example content. */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  private List<String> comments = new ArrayList<String>(
+    Arrays.asList("Nice!",
+                  "Very interesting.",
+                  "I would like to know more about your favourite TV shows!"));
+
+  /**
+   * Converts a List instance into a JSON string using manual String concatentation.
```

Comment:
A javadoc comment should document the api and not the implementation details. Therefore, remove the last part about how it converts.

Reply:
For now and a first draft it’s on to just build this using string concatenation but you should look into using a builtin java api or library to do json serialization and deserialization. I think there should be something in javax

Reply:
Otherwise there is eg gson which is a Google library for json serialization and deserialization

Reply:
Yes, they showed us the gson library and I will use it for the next steps.